apps/accounts/templates/accounts/change_secret_report.html

```diff
@@ -0,0 +1,314 @@
+        font-weight: 500;
+        font-size: 1.5rem;
+    }
+
+    .main-section .synopsis-section .synopsis-item .synopsis-item-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+        align-items: flex-start;
+        height: 100%;
+        cursor: pointer;
+    }
+
+    .main-section .synopsis-section .synopsis-item .synopsis-item-content p {
+        display: inline-flex;
+        width: 100%;
+        line-height: 1;
+        gap: 2rem;
+        font-size: 14px;
+    }
+
+    .main-section .synopsis-section .synopsis-item .synopsis-item-content p .item-label {
+        width: 18rem;
+    }
+
+    .main-section .synopsis-section .synopsis-item .synopsis-item-content span {
+        align-items: center;
+        justify-content: center;
+        font-weight: normal;
+    }
+
+    .main-section .tabel-summery-section {
+        display: flex;
+        flex-direction: column;
+        flex-wrap: nowrap;
+        align-items: flex-start;
+        gap: 2rem;
+        width: inherit;
+        margin-top: 2rem;
+    }
+
+    .main-section .tabel-summery-section .result-section {
+        width: 100%;
+        padding: 1rem;
+        background: #fff;
+        border: 1px solid #e0e0e0;
+        border-radius: 8px;
+    }
+
+    .main-section .tabel-execution-section h3 {
+        font-size: 1.5rem;
+        color: #2c3e50;
+    }
+
+    .section-header {
+        padding-bottom: 0.5rem;
+        margin-bottom: 1rem;
+        font-size: 1.5rem;
+    }
+
+    .section-header h3 {
+        display: flex;
+        align-items: center;
+        gap: 0.5rem;
+        margin: 0;
+        color: #2c3e50;
+        font-size: inherit;
+    }
+
+    .section-header span {
+        background: #e8f5e9;
+        color: #2e7d32;
+        padding: 0.2rem 0.8rem;
+        border-radius: 1rem;
+        font-size: inherit;
+    }
+
+    .custom-table {
+        margin-bottom: 0;
+        width: 100%;
+        border-collapse: collapse;
+        overflow: hidden;
+    }
+
+    .custom-table th {
+        background: #f6f6f6;
+        color: #495057;
+        padding: 0.75rem;
+        font-size: 1.5rem;
+        border-bottom: 2px solid #e0e0e0;
+    }
+
+    .custom-table td {
+        padding: 0.75rem;
+        font-size: 14px;
+        border-bottom: 1px solid #e0e0e0;
+    }
+
+    .custom-table tr:nth-child(even) {
+        background-color: #f6f6f6;
+    }
+
+    .custom-table tr:last-child td {
+        border-bottom: none;
+    }
+
+    .no-data {
+        text-align: center;
+        color: #6c757d;
+        padding: 2rem;
+        background: #f6f6f6;
+        border-radius: 8px;
+        margin: 1rem 0;
+        box-shadow: 0 1px 3px rgba(0, 0, 0, 0.1);
+    }
+
+    .new-accounts .section-header span {
+        background: #e8f5e9;
+        color: #2e7d32;
+    }
+
+    .lost-accounts .section-header span {
+        background: #fbe9e7;
+        color: #d84315;
+    }
+</style>
```

Comment:
Here are some key points to consider:

- The use of relative import statements such as `from django.utils import timezone` in your code could potentially confuse other developers who may be looking at this documentation.

- In general, Django's `timezone.now()` function should already return an UTC timestamp that can be used seamlessly without the need for relative imports. 

- This might not significantly impact performance though; it depends on how long each call takes to get the time.
  
- As per our current knowledge cutoff (September 2021), there does not seem to be anything particularly unique regarding syntax which could pose a major risk for bugs moving forward. However, keep up with Python standards and best practices going forward!